x-pack/plugins/apm/public/components/app/settings/agent_explorer/agent_instances/agent_contextual_information/index.tsx

```diff
@@ -0,0 +1,119 @@
+import { ApmRoutes } from '../../../../../routing/apm_route_config';
+import { ServiceLink } from '../../../../../shared/service_link';
+import { StickyProperties } from '../../../../../shared/sticky_properties';
+import { TruncateWithTooltip } from '../../../../../shared/truncate_with_tooltip';
+import { AgentExplorerDocsLink } from '../../agent_explorer_docs_link';
+
+function formatString(value?: string | null) {
+  return value || NOT_AVAILABLE_LABEL;
+}
+
+export function AgentContextualInformation({
+  agentName,
+  serviceName,
+  agentDocsPageUrl,
+  instances,
+  query,
+}: {
+  agentName?: AgentName;
+  serviceName?: string;
+  agentDocsPageUrl?: string;
+  instances?: number;
+  query: TypeOf<ApmRoutes, '/settings/agent-explorer'>['query'];
+}) {
+  const stickyProperties = [
+    {
+      label: i18n.translate('xpack.apm.agentInstancesDetails.serviceLabel', {
+        defaultMessage: 'Service',
+      }),
+      fieldName: AgentExplorerFieldName.ServiceName,
+      val: (
+        <TruncateWithTooltip
+          data-test-subj="apmAgentExplorerListServiceLink"
+          text={formatString(serviceName)}
+          content={
+            <ServiceLink
+              agentName={agentName}
+              query={{
+                kuery: query.kuery,
+                serviceGroup: '',
+                rangeFrom: 'now-24h',
+                rangeTo: 'now',
+                environment: query.environment,
+                comparisonEnabled: true,
+              }}
+              serviceName={serviceName ?? ''}
+            />
+          }
+        />
+      ),
+      width: '25%',
+    },
+    {
+      label: i18n.translate('xpack.apm.agentInstancesDetails.agentNameLabel', {
+        defaultMessage: 'Agent Name',
+      }),
+      fieldName: AgentExplorerFieldName.AgentName,
+      val: (
+        <EuiFlexGroup alignItems="center" gutterSize="s" responsive={false}>
+          <EuiFlexItem className="eui-textTruncate">
+            <span className="eui-textTruncate">{agentName}</span>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      ),
+      width: '25%',
+    },
+    {
+      label: i18n.translate('xpack.apm.agentInstancesDetails.intancesLabel', {
+        defaultMessage: 'Instances',
+      }),
+      fieldName: 'instances',
+      val: (
+        <EuiFlexGroup alignItems="center" gutterSize="s" responsive={false}>
+          <EuiFlexItem className="eui-textTruncate">
+            <span className="eui-textTruncate">{instances}</span>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      ),
+      width: '25%',
+    },
+    {
+      label: i18n.translate(
+        'xpack.apm.agentInstancesDetails.agentDocsUrlLabel',
+        {
+          defaultMessage: 'Agent documentation',
+        }
+      ),
+      fieldName: AgentExplorerFieldName.AgentDocsPageUrl,
+      val: (
+        <TruncateWithTooltip
+          data-test-subj="apmAgentExplorerListDocsLink"
+          text={formatString(`${agentName} agent docs`)}
```

Comment:
What's the point of wrapping this in `formatString`? you're always passing a non-falsy string anyway no? (Preferably these props are just required)

Reply:
There is an agent name (`otlp`) that doesn't have a docs url

Reply:
Yes, I just saw your point. Deleting the function from the code in the next commit

Reply:
Yes, I was talking about `AgentExplorerDocsLink` 😅